.gitignore

```diff
@@ -18,13 +18,3 @@ out
 *.orig
 *.rej
 *.bak
-*.kate-swp
```

Comment:
This is reverting changes I made upstream to ignore the `.vscode` directory and other unwanted files.

Reply:
oh, well ok then